Aspire BudgetingTests/MockUser.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MockUser.swift
+//  Aspire BudgetingTests
+//
+//  Created by TeraMo Labs on 11/1/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+@testable import Aspire_Budgeting
+import Foundation
+import GTMSessionFetcher
+
+class MockUser: AspireUser {
+  typealias Profile = MockProfile
+  
+  typealias Authentication = MockAuthentication
+  
+  var profile: Profile! = MockProfile()
+  var authentication: Authentication! = MockAuthentication()
+  
+}
+
+class MockProfile: AspireProfile {
+  var name: String! = "First Last"
+}
+
+class MockAuthentication: AspireAuthentication {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/6ead50c19f54008e63b8e9f2ba720e23a9263087/Aspire BudgetingTests/MockUser.swift#L27"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>